templates/myaccount/dashboard.php

```diff
@@ -1,11 +1,11 @@
 <?php
 /**
- * My Account page
+ * My Account page.
  *
  * @package LifterLMS/Templates
  *
  * @since    1.0.0
- * @version  3.14.0
+ * @version  [version]
```

Comment:
can you please do?
```
 *
 * @since 1.0.0
 * @since 3.14.0 Unknown.
 * @version [version]
 ````
 please note the spacing too.

Reply:
updated.